tests/unit/handlers/test_structured_log.py

```diff
@@ -207,6 +207,52 @@ def test_format_with_custom_formatter(self):
         self.assertIn(expected_result, result)
         self.assertIn("message", result)
 
+    def test_format_with_reserved_json_field(self):
+        # drop json_field data with reserved names
+        # related issue: https://github.com/googleapis/python-logging/issues/543
+        import logging
+        import json
+
+        handler = self._make_one()
+        message = "hello world"
+        extra = "still here"
+        json_fields = {
```

Comment:
Is this supposed to be case sensitive? I would add extra tests for fields with capital letters...